### DISCUSSION TITLE
Bottom Up Message Order

### DISCUSSION THREAD

**Discussion by gaussandhisgun** [Chatterino/chatterino2#4954] [Ideas]
So Twitch messages usually appear on the bottom of the message list. But after watching *specifically Matthew Judge* for a few streams, I've noticed he uses some UserCSS on his Twitch in Chrome so he gets the new messages on the top instead. Can a feature similar to this be implemented in Chatterino?

**Comment by Felanbird** [comment]
This has been denied before because minimal change to CSS is literally nothing compared to what we would need to do to implement this into Chatterino. https://github.com/Chatterino/chatterino2/issues/2821#issuecomment-847554150

**Comment by Mm2PL** [comment]
Chatterino is not a browser and thus cannot use CSS. We have custom layout code that would need a lot of changes to allow for this.

**Comment by Mm2PL** [comment]
It may sound stupid, but consider making an upside-down font and flipping the window somehow. Or make a patch for this if you have the knowledge to do so.
